src/PackageManagement.VisualStudio/IDE/EnvDTEProjectName.cs

```diff
@@ -1,49 +1,49 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

```

Comment:
Can you please fix the line-ending diff that shows up in this file and all the other files? This shows up as a diff, even though it is unrelated to your fix